Products/PluggableAuthService/plugins/ScriptablePlugin.py

```diff
@@ -15,8 +15,11 @@
 
 $Id$
 """
-from sets import Set
-from urllib import quote_plus
+try:
+    from sets import Set as set
+except ImportError:
+    pass
+
```

Comment:
Just remove the import and *always* use the built-in `set` type.

Reply:
Done.